spec/spec_helper.rb

```diff
@@ -1,4 +1,6 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'simplecov'
+SimpleCov.start
 
 if ENV["TRAVIS"]
```

Comment:
```suggestion
if ENV["TRAVIS"] || ENV["COVERAGE"]
```

Instead of adding the extra block, can we just tap into the existing simple coverage entry?
Either use `CI=true`, `COVERAGE=true`, or `SIMPLECOV=true` - which ever works best for your